web/site.webmanifest

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "",
-  "short_name": "",
```

Comment:
Why were these removed?

Reply:
I changed their values to `"Frigate"` instead of leaving them blank.